docs/descriptor/packages.rst

```diff
@@ -19,13 +19,35 @@ Packages are defined in the ``install`` subsection.
 ``repositories``
 ----------------
 Cekit uses all repositories configured inside the image. You can also specify additional
-repositories using repositories subsection. Cekit currently supports three ways of defining
+repositories using repositories subsection. Cekit currently supports four ways of defining
 additional repositories:
 
+* Plain
 * RPM
 * ODCS
 * URL
 
+.. note::
+   See :ref:`Repository mangement<repository_management>` to learn about best practices for repository
+   definitions.
+
+.. _repo_plain:
+
+``Plain``
+^^^^^^^^^
+This is the default option. With this approach you specify repository id and Cekit will not perform any action and expect the repository definition exists inside the image.
```

Comment:
Add more information:

* this is just a hint what repositories should be provided
* explain main use case - to be overridden if needed by actual definition found below.